product/tests/test_product.py

```diff
@@ -16,20 +17,34 @@ def product0():
     return Product(qr_code=QR, product_name=NAME, description=DESCRIPTION)
 
 
+@pytest.fixture
+def product1():
+    return Product(qr_code=QR1, product_name=NAME, description=DESCRIPTION)
+
+
 @pytest.fixture
 def saved_product0(product0):
     product0.save_product()
     return product0
 
 
+@pytest.fixture
+def saved_product1(product1):
+    product1.save_product()
+    return product1
+
+
 class TestProductModel:
     @pytest.mark.django_db()
-    def test_save_product_and_delete_product(self, saved_product0):
+    def test_save_product_and_delete_product(self, saved_product0, saved_product1):
```

Comment:
I'll let it slide, but do note that tests usually should check for one thing (e.g: a test for saving a product and a test for deleting a product). then, if a test failed, you can pinpoint where the issue lies more easily.

Reply:
This was my suggestion in this case. @thetaxas2468 proposed separate tests, but I suggested combining them for this purpose as it made sure that we add a product that doesn't exist, and that we delete a product that does exist.

Reply:
I have done that before